astronomer/providers/amazon/aws/triggers/sagemaker.py

```diff
@@ -189,6 +189,9 @@ class SagemakerTrainingWithLogTrigger(BaseTrigger):
     """
     SagemakerTrainingWithLogTrigger is fired as deferred class with params to run the task in triggerer.
 
+    This class is deprecated and will be removed in 2.0.0.
```

Comment:
Let's also add a warning in the init method?

Reply:
Sure. missed this one